impl/restart_integration_test.go

```diff
@@ -31,6 +32,19 @@ type peerError struct {
 	err error
 }
 
+var monitorConfig = channelmonitor.Config{
+
+	AcceptTimeout:          time.Hour * 24,
+	RestartDebounce:        time.Second * 10,
+	RestartBackoff:         time.Second * 20,
+	MaxConsecutiveRestarts: 15,
+	//RestartAckTimeout:      time.Second * 30,
```

Comment:
Seems like this shouldn't be here, it's not in `Config` type. Is it a TODO for future work?

The `OnRestartComplete` below could probably be removed too since it's not being used, or is that another TODO?